org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_memberships.scss

```diff
@@ -55,6 +55,15 @@
 }
 
 .page-civicrm-contact-view {
+  .ui-widget-content {
+    &.ui-tabs-panel {
+      padding-right: 15px !important;
```

Comment:
but specific it like that won't work on all different screens without using media queries , right ?